Gemfile

```diff
@@ -53,7 +54,7 @@ gem "rack-cors", "0.4.0", require: "rack/cors"
 
 # CSS
 
-gem "bootstrap-sass", "2.3.2.2"
+gem "bootstrap-sass", "~> 3.3.4"
```

Comment:
Why that ? BS uses ssemver. This mean that the gem will always be updated with the last fixes. I think this is more secure.